libs/designsystem/item/src/item.component.html

```diff
@@ -1,9 +1,8 @@
 <ion-item
   lines="none"
   [attr.disabled]="disabled"
```

Comment:
Consider also handling this inprovement in this PR, as it seems tightly coupled to the button-functionality.
```suggestion
  [disabled]="disabled"
```

Reply:
There are lot of other stuff related to the disabled which I would like to handle in a separate PR 🤞🏻

Reply:
Good call!